.eleventy.js

```diff
@@ -65,35 +64,32 @@ module.exports = function (eleventyConfig) {
     }
   );
 
+  /** Converts a Date to human-readable string,
+   * e.g. 05 Oct 2021
+   */
+  const dateTimeFormatter = new Intl.DateTimeFormat('en-GB', {
+    timezone: "UTC",
+    day: "numeric",
+    month: "short",
+    year: "numeric",
+  });
+
+  /** Adds a leading zero to any number < 10 */
+  const numberFormatter = new Intl.NumberFormat('en-GB', {
+    minimumIntegerDigits: 2,
+});
+
   eleventyConfig.addFilter("readableDate", (dateObj) => {
-    return DateTime.fromObject(
-      {
-        year: dateObj.getFullYear(),
-        month: dateObj.getMonth() + 1,
-        day: dateObj.getDate(),
-      },
-      { zone: "utc" }
-    ).toFormat("dd LLL yyyy");
+    return dateTimeFormatter.format(dateObj);
   });
 
   // https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#valid-date-string
   eleventyConfig.addFilter("htmlDateString", (dateObj) => {
-    return DateTime.fromObject(
-      {
-        year: dateObj.getFullYear(),
-        month: dateObj.getMonth() + 1,
-        day: dateObj.getDate(),
-      },
-      { zone: "utc" }
-    ).toFormat("yyyy-LL-dd");
+    return dateObj.toISOString().replace(/(T.*$)/, '')
   });
 
-  // Reverse the index (1-based) and pad with leading zeroes
-  eleventyConfig.addFilter("reverseAndPad", function (index, list_length) {
-    return String(list_length + 1 - index).padStart(
-      Math.max(String(list_length).length, 2),
-      "0"
-    );
+  eleventyConfig.addFilter("minimumTwoDigits", (number) => {
+    return numberFormatter.format(number);
   });
```

Comment:
3️⃣ I really liked this filter, because it shows that you are good at thinking about Real Algorithmic Problems™ in a way that I very much am not. Fortunately, `numberFormatter` makes it pretty easy to pad a number with a zero. See 4️⃣  for other relevant changes.